src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -10012,6 +10016,9 @@ namespace OutputReportTabular {
 				if ( Zone( iZone ).FloorArea > 0 && usezoneFloorArea ) {
 					tableBody( 12, iZone ) = RealToStr( totPlugProcess * Wm2_unitConv / Zone( iZone ).FloorArea, 4 );
 				}
+				// Hybrid Model Multiplier Added by Sang Hoon Lee August 2015
+				HMMultiplier = Zone( iZone ).ZoneVolCapMultpSensHMAverage;
```

Comment:
We added a new table, Hybrid Model: Internal Thermal Mass instead of adding optional columns. The table is only shown when hybrid model: internal thermal mass is flagged.  It has two columns, the first column indicates Hybrid Modeling (Y/N) and the second column provides Temperature Capacitance Multiplier values for each zone.